elasticsearch/run.sh

```diff
@@ -25,20 +34,20 @@ if [ -z $(curl -s -f 'http://localhost:9200/searchguard/ac/ac') ]; then
       },
       {
         "__Comment__": "This is so that fluentd can only write",
-        "users": ["fluentd"],
+        "users": ["system.logging.fluentd"],
         "filters_bypass": [],
         "filters_execute": ["actionrequestfilter.fluentd"]
       },
       {
         "__Comment__": "This is so that Kibana can do anything in the .kibana index",
-        "users": ["kibana"],
-        "indices": [".kibana"],
+        "users": ["system.logging.kibana"],
```

Comment:
No since this one specifies an index that this is for (".kibana*").  The other one will be used for every other index